docs/en/sql-reference/functions/null-functions.md

```diff
@@ -52,6 +54,45 @@ Result:
 └───┘
 ```
 
+## isNullable
+
+Returns `1` if a column allows null values, `0` otherwise.
```

Comment:
`[...] if a column is Nullable (i.e. allows NULL values), [...]`

^^ please link `Nullable` to the doc page of the respective type